src/Gui/DlgGeneralImp.cpp

```diff
@@ -462,4 +536,23 @@ void DlgGeneralImp::onLoadPreferencePackClicked(const std::string& packName)
     }
 }
 
+
+void DlgGeneralImp::on_comboBox_UnitSystem_currentIndexChanged(int index)

```

Comment:
This style of function name makes it look like an auto-connected slot. If it is, it should not be: Clazy will complain about it, that use is no longer recommended. (Sorry, on a phone, it's hard to see for sure).

Reply:
In this PR I only copy pasted without having a more indepth look.
It appears that it is indeed auto-connected. I can change that.